src/modules/billing/mappers/api/invoice.js

```diff
@@ -9,17 +10,26 @@ const containsWaterUndertaker = invoiceLicences => {
     .includes(true);
 };
 
+const doesMinimumChargeApply = invoiceLicences => {
+  const transactions = flatMap(invoiceLicences.map(invoiceLicence => invoiceLicence.transactions));
+  return transactions.some(trans => trans.isMinimumCharge);
+};
+
 const map = {
   id: 'id',
   'invoiceAccount.accountNumber': 'accountNumber',
   'invoiceAccount.company.name': 'name',
   'totals.netTotal': 'netTotal',
   'invoiceLicences[].licence.licenceNumber': 'licenceNumbers[]',
   'financialYear.yearEnding': 'financialYearEnding',
-  invoiceLicences: {
+  invoiceLicences: [{
     key: 'isWaterUndertaker',
     transform: containsWaterUndertaker
+  }, {
+    key: 'minimumChargeApplies',
```

Comment:
We generally have a convention of starting the property name with `is` for a boolean e.g. `isMinimumChargeApplied`